### ISSUE TITLE
"Your current platform does not support automatic detection"

### ISSUE THREAD

**Issue by pwflint** [eth-p/obsidian-callout-manager#8]
Some screenshots below of a warning/error message received after install. Plug-in seems to be working as expected.

Debug:

SYSTEM INFO:
	Obsidian version: v1.3.7
	Installer version: v1.3.5
	Operating system: Darwin Kernel Version 21.6.0: Sat Jun 18 17:07:22 PDT 2022; root:xnu-8020.140.41~1/RELEASE_ARM64_T6000 21.6.0
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: adapt to system
	Community theme: Everforest v0.0.0
	Snippets enabled: 4
	Restricted mode: off
	Plugins installed: 26
	Plugins enabled: 24
		1: Book Search v0.4.0
		2: Advanced Codeblock v1.1.5
		3: Excalidraw v1.7.15
		4: Activity History v0.1.4
		5: Dataview v0.5.41
		6: Image Gallery v1.0.0
		7: Zotero Integration v2.2.24
		8: Things Logbook v0.1.16
		9: Templater v1.14.3
		10: Kanban v1.4.5
		11: Projects v1.11.5
		12: Waypoint v1.3.0
		13: Highlightr v1.2.2
		14: Periodic Notes v0.0.17
		15: Natural Language Dates v0.6.1
		16: Banners v1.3.3
		17: Matter v1.1.4
		18: Tray v0.3.3
		19: Obsidian Git v2.10.0
		20: Tasks v4.2.0
		21: Homepage v3.3.1
		22: Buttons v0.4.19
		23: Calendar v1.5.10
		24: Callout Manager v1.0.1

<img width="640" alt="Screen Shot 2023-08-07 at 16 25 28" src="https://github.com/eth-p/obsidian-callout-manager/assets/73616669/2b69b838-01b9-47f7-95b1-b53c49e8c050">
<img width="759" alt="Screen Shot 2023-08-07 at 16 25 19" src="https://github.com/eth-p/obsidian-callout-manager/assets/73616669/94f663e5-8f1b-4391-b406-960b7ac1782f">

**Comment by legolasdimir** [comment]
I am seeing the same message. I think its causing a custom callout from another plugin I'm using to not show up in the manage menu.

**Comment by KrisKimball** [comment]
Same issue, and I'm also unable to manage callouts - app freezes while on the manage callout page, and I have to quit and reopen. Disabled the plugin and looking for other solutions.

**Comment by mathisgauthey** [comment]
Yup, there's no autocompletion on my side as well. Strangely, the callouts type by default are correctly found in the plugin settings.

**Comment by Zorgonaute84** [comment]
Double check if you don't have custom callout including uppercase letter Chears ;-)

**Comment by zenus260** [comment]
> Same issue, and I'm also unable to manage callouts - app freezes while on the manage callout page, and I have to quit and reopen. Disabled the plugin and looking for other solutions.

You actually don't have to quit and reopen. Just press escape. It seems like the app doesn't freeze, that menu just can't be interacted with.

**Comment by eth-p** [comment]
I'm happy to say that the now-published version 1.1.0 should no longer have this problem, as it uses standard functions to find the Obsidian callouts instead of using internal/undocumented Obsidian ones.

> Same issue, and I'm also unable to manage callouts - app freezes while on the manage callout page,

I'm not entirely sure why that would happen, but please let me know if it still does it with the new version.
